blog/_posts/2024-06-25-history-ele-raid-buffs.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: The Historical Precedent for an Elemental Raid Buff
+#            dd/mm/yyyy
+last_update: 25/06/2024
+excerpt: Elemental Shaman's place as a raid buff provider.
+author: Lucenia
+toc: true
+big_article: true
+---
+
+> Lucenia is a 15-year veteran of Elemental Shaman currently Cutting-Edge raiding in [\<Noted\>](https://raider.io/guilds/us/akama/Noted) on US-Akama. She streams mythic raiding, mythic+, and PTR/beta testing content at https://twitch.tv/Lucenia.
+She also produces sims, guides, and other content for Elemental Shaman in Cataclysm Classic and Season of Discovery.
```

Comment:
In case you don't want this sentence to be part of the quote block you need to add another empty line above. In case you want it to be part of the quote block, please add `>` in front of it to make the intention clear.